src/lib/validators.ts

```diff
@@ -1,28 +1,31 @@
-'use strict';
 import * as Bluebird from 'bluebird';
 import { stat } from 'fs';
 import { isFQDN, isPort } from 'validator';
 
 const statAsync = Bluebird.promisify(stat);
```

Comment:
`mz/fs`.

Reply:
Validators are dropped, as we are now using schema everywhere.